test/models/v3/getPenaltyDetails/FailureCodeEnumSpec.scala

```diff
@@ -17,6 +17,7 @@
 package models.v3.getPenaltyDetails
 
 import base.SpecBase
+import models.v3.FailureCodeEnum
 import play.api.libs.json.{JsString, Json}
```

Comment:
This also needs to be moved to v3, as it's been moved from /v3/getPenaltyDetails